thundra/wrappers/flask/flask_executor.py

```diff
@@ -1,17 +1,30 @@
 from thundra import constants
+from thundra.config import config_names
+from thundra.config.config_provider import ConfigProvider
 
 from thundra.wrappers import wrapper_utils, web_wrapper_utils
 
 
 def start_trace(plugin_context, execution_context, tracer):
     request = execution_context.platform_data['request']
     request_route_path = str(request.url_rule) if request.url_rule else None
+    '''
+        request data read as stream. This function cache the read data and serve from cache.
+        According to function comment in Flask, content length should be checked before read!!! 
+    '''
+    req_data = None
+    if not ConfigProvider.get(config_names.THUNDRA_TRACE_REQUEST_SKIP, False):
+        cl = request.content_length
+        if cl == None or cl <= constants.THUNDRA_MAX_STREAM_REQUEST_BODY:
+            req_data = request.get_data()
+        else:
```

Comment:
it might be better to skip capturing request data completely if it is bigger than limit instead of getting it partially

Reply:
done.